techniques/failures/F50.html

```diff
@@ -123,18 +117,18 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
    </section>
 
 
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../client-side-script/SCR22">SCR22: Using scripts to control blinking and stop it in five seconds or less</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
-         <p>For each instance of blinking content:</p>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
+         <p>点滅しているそれぞれの箇所に対して:</p>
          <ol>
-            <li>Determine if the blinking stops in 5 seconds or less.</li>
+            <li>点滅が 5 秒以内で止まるかどうかをチェックする。</li>
```

Comment:
原文が check ではなく determine なので、
https://github.com/waic/wcag22/pull/1193
に合わせて、「判断する」とするのはいかがでしょうか。
```suggestion
            <li>点滅が 5 秒以内で止まるかどうかを判断する。</li>
```